### DISCUSSION TITLE
Does changing node version have any impact on lmdb?

### DISCUSSION THREAD

**Discussion by maghaffar** [kriszyp/lmdb-js#275] [General]
I have changed node js version from 16.14.0 to 20.11.1 to meet the requirements of Gatsby 5 and now I am getting error when installing dependencies using yarn install.
The error is with lmdb-store : **/opt/build/repo/node_modules/lmdb-store: Command failed.**
error: no matching function for call to ‘v8::Symbol::Description()’
Local<String> string = Local<String>::Cast(Local<Symbol>::Cast(jsKey)->Description());
I have tried a lot of solutions from internet but none of them worked
I am also attaching ss of my error
![Screenshot from 2024-02-28 17-45-07](https://github.com/kriszyp/lmdb-js/assets/98881071/bb09a3db-454a-4e2d-bf0d-80641e5de049)

**Comment by kriszyp** [comment]
I believe this has been fixed for a quite a while in lmdb-js, at least since 2.3.

**Comment by maghaffar** [comment]
the reason of the error was that I was using lmdb-store which is now deprecated. When I changed the package and installed lmdb, the error was gone.
